includes/admin/views/tools/danger-zone.php

```diff
@@ -0,0 +1,57 @@
+<div class="erp-grid-container tools-section erp-danger-zone">
+    <h3 class="section-title"><?php esc_html_e('Reset WP ERP', 'erp'); ?></h3>
+
+    <div class="tools-alert tools-alert-warning">
+        <h4 class="tools-alert-title">
+            <i class="fa fa-warning"></i>
+            <?php esc_html_e('Warning', 'erp'); ?>
+        </h4>
+        <p><?php _e('The reset makes a fresh installation of your database. Therefore any data related to Wp ERP in your database will be lost. Please do not use this option if you do not want to delete all of your WP ERP data.', 'erp'); ?></p>
```

Comment:
Considering WP coding standard, there should be a space after the opening parentheses and before the closing parentheses while using a function, `i.e., _e( 'Reset WP ERP', 'erp' )`